tests/core/core-to-snapd-failover16/task.yaml

```diff
@@ -0,0 +1,55 @@
+summary: Test the failover scenario of the snapd snap installation on a UC16 system
+
+# snapd snap is already installed by default on uc18+
+systems: [ubuntu-core-16-*]
+
+prepare: |
+  # shellcheck source=tests/lib/snaps.sh
+  . "$TESTSLIB/snaps.sh"
+  repack_core_snap_into_snapd_snap
+
+execute: |
+  # shellcheck source=tests/lib/journalctl.sh
+  . "$TESTSLIB/journalctl.sh"
+
+  echo "Re-pack snapd with broken snapd"
+  unsquashfs -d ./snapd-broken snapd-from-core.snap
+  echo "" > ./snapd-broken/usr/lib/snapd/snapd
+  # pick up snap-failure from the current snapd
+  (cd ./snapd-broken && snap pack .)
+
+  # try to install it multiple times to ensure that we can revert back to the 
+  # core snap as many times as needed, if a device for some reason keeps wanting
+  # to try to transition to the snapd snap and keeps failing
+  #shellcheck disable=SC2167
+  for _ in $(seq 1 2); do
+    # get the number of times that snapd.failure was started before we do 
+    # anything
+    started_before="$(get_journalctl_log -u snapd.failure | grep -c 'Started Failure handling of the snapd snap.' || true)"
+
+    echo "Now install the broken snapd"
+    if snap install --dangerous ./snapd-broken/snapd_*.snap; then
+        echo "installing a broken snapd should not work, test broken"
```

Comment:
Technically the test has failed, it doesn't imply the test is broken, perhaps snapd is broken.

Reply:
sure, I can re-word the message here